documentation/modules/oauth/con-oauth-authorization-keycloak-example.adoc

```diff
@@ -100,21 +136,21 @@ Roles cannot be stored in an LDAP identity provider.
 If LDAP is a requirement, you can use groups instead, and add Keycloak roles to the groups so that when users are assigned a group, they also get a corresponding role.
 
 Under *Clients*, you can view the additional client configurations. For this example,  `kafka`, `kafka-cli`, `team-a-client`, `team-b-client` are configured.
```

Comment:
To scan easily, maybe we can list all the example configs we can see in the console in a bulleted list? (A table of all the configs would work too.)

Reply:
This is not an exhaustive list, it's meant to be informational, not a matrix of choices.